src/index.ts

```diff
@@ -139,6 +158,13 @@ export default class NewRelicLambdaLayerPlugin {
       return;
     }
 
+    if (this.config.enableExtension && this.licenseKey) {
```

Comment:
Note that if they want function logs sent to NR, we still need the log subscription, with the "" filter.

Reply:
Yeah, there's an `logsEnabled` config option that should decide if logs ingestion via CLoudWatch is enabled or not.

Reply:
Would the decision ride solely on the logEnabled config var? If that's the case, then I could just rip out this conditional return and let the usual log subscription setup handle it.